proto/viam/app/data/v1/data.proto

```diff
@@ -256,15 +256,15 @@ message DeleteBinaryDataByIDsResponse {
   reserved "result";
 }
 
-// AddTagsToBinaryDataByFileIDsRequest requests adding all specified tags to each of the files when binary ids are provided
-message AddTagsToBinaryDataByFileIDsRequest {
+// AddTagsToBinaryDataByIDsRequest requests adding all specified tags to each of the files when binary ids are provided
+message AddTagsToBinaryDataByIDsRequest {
   repeated BinaryID binary_ids = 3;
   repeated string tags = 2;
 
   repeated string file_ids = 1 [deprecated = true]; // Replaced by binary_ids
```

Comment:
Do we want to reserve instead of deprecate?

Reply:
Yep doing that in a separate PR since requires a more involved RDK fix.